### ISSUE TITLE
Finalize system architecture within target hosting environments

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-cloud-migration#6]
This task is the next phase after #5 where we create a new revision of the system architecture diagram that identifies the specific hosting environment and container technologies each component will use. This might include RDS, Docker Swarm, Cloud Foundry, etc

**Comment by JJediny** [comment]
Current plan is to use a hybrid approach to deploy Data.gov and Labs.data (php based wordpress and project open data dashboard) through Cloud Foundry... The main CKAN application(s) in catalog.data.gov and inventory.data.gov would be deployed along side cloud foundry in the same manner the various services offered through cf are deployed/managed using http://bosh.io supplementing/appending our proposed bosh deployments to 18F's bosh deployments via:

* https://github.com/18F/cg-manifests
* https://github.com/GSA/cg-manifests

In short we are deploying our application by the same means/method services are deployed for cloud foundry...

### BOSH process:
![image](https://cloud.githubusercontent.com/assets/6350035/12485263/01761be4-c02c-11e5-9014-afa29c8e015e.png)

### Background:
* https://bosh.io/docs/bosh-components.html
* https://blog.pivotal.io/pivotal-cloud-foundry/products/managing-stateful-docker-containers-with-cloud-foundry-bosh
